python/rbac/utils.py

```diff
@@ -15,11 +15,16 @@
 from django.db.models import Model
 from rest_framework import serializers
 
+from cm.errors import AdcmEx
+
 
 class BaseRelatedSerializer(serializers.Serializer):
     def to_internal_value(self, data):
         data = super().to_internal_value(data)
-        return data['id']
+        try:
+            return data["id"]
+        except KeyError as e:
+            raise AdcmEx("INVALID_DATA", f"Field {e} is required") from e
```

Comment:
```suggestion
        if "id" not in data:
            raise serializers.ValidationError("This field may not be empty.")
        return data.get("id")
```
`is required` invalid error for parent class

Reply:
done